src/Adapters/Storage/Mongo/MongoTransform.js

```diff
@@ -330,18 +330,11 @@ const parseObjectToMongoObjectForCreate = (className, restCreate, schema) => {
 // Main exposed method to help update old objects.
 const transformUpdate = (className, restUpdate, parseFormatSchema) => {
   let mongoUpdate = {};
-  let acl = addLegacyACL(restUpdate)._acl;
-  if (acl) {
-    mongoUpdate.$set = {};
-    if (acl._rperm) {
-      mongoUpdate.$set._rperm = acl._rperm;
-    }
-    if (acl._wperm) {
-      mongoUpdate.$set._wperm = acl._wperm;
-    }
-    if (acl._acl) {
-      mongoUpdate.$set._acl = acl._acl;
```

Comment:
If it was written like that, there must be a reason, that is not obvious. The `acl._acl` is clearly an error from the original commiter and not his intention. I'd like to keep the code as it is, as it's working for the other use cases. You don't really know what you maybe breaking by removing it